pegasus/migrations/105_add_contacts_rollup_table.rb

```diff
@@ -0,0 +1,29 @@
+Sequel.migration do
+  change do
+    create_table(:contacts_rollup, charset: 'utf8') do
+      primary_key :id, unsigned: true, null: false
+      String :email
+      Int :dashboard_user_id
+      String :name
+      String :street_address, size: 1024
+      String :city
+      String :state
+      String :country
+      String :postal_code
```

Comment:
Do these address fields refer to the user or to the school? Probably worth at least adding a comment to the migration file, perhaps prefacing the field names with `user` or `school`.